Tools/clinic/clinic.py

```diff
@@ -5266,6 +5263,11 @@ def state_parameter_docstring_start(self, line: str) -> None:
 
     def docstring_append(self, obj: Function | Parameter, line: str) -> None:
         """Add a rstripped line to the current docstring."""
+        if match := re.search(r'[^\x00-\x7F]', line):
```

Comment:
(Optional:) You could use `re.finditer` if you wanted to iterate over all matches

Reply:
Yes, that's even better.